src/protocols/1/native-protocols/mqtt/index.md

```diff
@@ -0,0 +1,185 @@
+}));
+
+// Getting Kuzzle's response
+client.on('message', (topic, raw) => {
+  const message = JSON.parse(Buffer.from(raw));
+
+  // API results topic
+  if (topic === 'Kuzzle/response') {
+    // Response to our "publish" request
+    if (message.requestId === 'some unique ID') {
+      console.log('Message publication result: ', message);
+    }
+  }
+});
+```
+
+### Using Kuzzle subscriptions
+
+Kuzzle allows to [subscribe](https://docs.kuzzle.io/api/1/controller-realtime/subscribe/) to messages and events using advanced filters.
+
+Each time a subscription is sent, a dedicated MQTT topic is created, named after the ``channel`` property issued by Kuzzle.
+
+Here are the steps to perform a Kuzzle subscription using MQTT:
+
+- Send a subscription request to Kuzzle
+- Listen to the request's response to ge the ``channel`` identifier
+- Subscribe to the MQTT topic named after this channel identifier
+
+Example using [MQTT Node module](https://www.npmjs.com/package/mqtt):
+
+```javascript
+const
+  mqtt = require('mqtt'),
+  client = mqtt.connect({host: 'localhost'}),
+  channels = [];
+
+// Sending a volatile message
+client.publish('Kuzzle/request', JSON.stringify({
+  index: 'index',
+  collection: 'collection',
+  controller: 'realtime',
+  action: 'subscribe',
+  requestId: 'some unique ID',
+  body: {
+    term: {
+      some: 'filter'
+    }
+  }
+}));
+
+// Getting Kuzzle's response
+client.on('message', (topic, raw) => {
+  const message = JSON.parse(Buffer.from(raw));
+
+  // API results topic
+  if (topic === 'Kuzzle/response') {
+    // Response to our "publish" request
+    if (message.requestId === 'some unique ID') {
+      channels.push(message.result.channel);
+      client.subscribe(message.result.channel);
+    }
+  }
+  else if (channels.indexOf(topic) !== -1) {
+    // Subscription notification
+    console.log('Notification: ', message);
+  }
+});
+```
+
+## Authorizations
+
+### Publishing
+
+If ``allowPubSub`` is set to ``false``, clients can only publish to the ``requestTopic`` topic (defaults to ``Kuzzle/request``).
+
+If ``allowPubSub`` is set to ``true``, clients are only forbidden to publish to the ``responseTopic`` topic (defaults to ``Kuzzle/response``).
+
+<div class="alert alert-warning">
+    Wildcards subcriptions are not allowed
+</div
+
+If a client tries to publich to an unauthorized topic, his connection will immediately be shut down by the server.
+
+### Subscribing
+
+Subscription attempts to the ``requestTopic`` topic (defaults to ``Kuzzle/request``) are ignored: client requests can only be listened by the MQTT server.
+
+## Development mode
+
+The MQTT ``Kuzzle/response`` topic is by default a special topic that acts as a private channel. Each client receives its own responses only, offering a simple first security layer.
+
+While this behaviour is urgently recommended in production, it can bring a small drawback when testing and developing applications: it does not allow using most CLI tools.  
```

Comment:
```suggestion
While this behavior is urgently recommended in production, it can bring a small drawback when testing and developing applications: it does not allow using most CLI tools.  
```